go.mod

```diff
@@ -67,7 +67,7 @@ require (
 	github.com/jbenet/go-temp-err-catcher v0.1.0
 	github.com/jbenet/goprocess v0.1.4
 	github.com/libp2p/go-doh-resolver v0.4.0
-	github.com/libp2p/go-libp2p v0.25.1
+	github.com/libp2p/go-libp2p v0.25.2-0.20230222220431-c03190e1ca4f
```

Comment:
:information_source:  switch to release after https://github.com/ipfs/kubo/pull/9656 lands.